sdk/Dagger.SDK.Mod.SourceGenerator.Tests/SourceGeneratorTests.cs

```diff
@@ -0,0 +1,100 @@
+using System.Collections.Immutable;
+using System.Reflection;
+
+using Basic.Reference.Assemblies;
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+
+using VerifyMSTest;
+
+using Binder = Microsoft.CSharp.RuntimeBinder.Binder;
+
+namespace Dagger.SDK.Mod.SourceGenerator.Tests;
+
+[TestClass]
+public class SourceGeneratorTests : VerifyBase
```

Comment:
I have discussion with Microsoft team in C# discord. They're recommended me to use https://www.nuget.org/packages/Microsoft.CodeAnalysis.CSharp.SourceGenerators.Testing#supportedframeworks-body-tab rather than `Verify`. I don't see much example from it but it would be nice if we strict with the Microsoft tools.

Reply:
This is what I thought I had added here: https://github.com/wingyplus/dagger-dotnet-sdk/blob/main/sdk/Dagger.SDK.SourceGenerator/Dagger.SDK.SourceGenerator.Tests/Dagger.SDK.SourceGenerator.Tests.csproj#L17